keras_cv/layers/preprocessing/random_choice.py

```diff
@@ -73,11 +80,16 @@ def call_layer():
 
         return call_layer
 
+    def _batch_augment(self, inputs):
+        if self.batchwise:
```

Comment:
Is this correct? If `batchwise` we don't call `_batch_augment`?

Reply:
yeah it is correct.  if you use batchwise you should pass everything to the same layer.